orttraining/orttraining/python/orttraining_pybind_state.cc

```diff
@@ -475,27 +480,26 @@ void addObjectMethodsForTraining(py::module& m) {
         return static_cast<PipelineTrainingSession*>(sess->GetSessionHandle())->IsGraphOutputFp32Node(output_name);
       });
 
-py::class_<TrainingAgent>(m, "TrainingAgent", R"pbdoc(This is the main class used to run a ORTModule model.)pbdoc")
+  py::class_<PartialGraphExecutionState>(m, "PartialGraphExecutionState")
+      .def(py::init([]() {
+        return onnxruntime::make_unique<PartialGraphExecutionState>();
+      }));
+
+  py::class_<TrainingAgent>(m, "TrainingAgent", R"pbdoc(This is the main class used to run a ORTModule model.)pbdoc")
       // In Python3, a Python bytes object will be passed to C++ functions that accept std::string or char*
       // without any conversion. So this init method can be used for model file path (string) and model content (bytes)
```

Comment:
Not part of this PR, but this comment seems obsolete (we don't pass a model file or content to this method).